docs/book/releases/v2.1.0.md

```diff
@@ -48,15 +48,18 @@ Note: For vSphere CSI Migration feature the minimum Kubernetes version requireme
       - If user has accidentally left orphan volumes on the datastore by not following the guideline, and if user has captured the volume handles or First Class Disk IDs of deleted PVs, storage admin can help delete those volumes using `govc disk.rm <volume-handle/FCD ID>` command.
 6. When in-tree vSphere plugin is configured to use default datastore in this format default-datastore: `</datacenter-name>/datastore/<default-datastore-name>`, migration of the volume will fail as mentioned here: https://github.com/kubernetes-sigs/vsphere-csi-driver/issues/628. If default datastore is configured in this format `<default-datastore-name>` then we do not see this issue
    - Impact: In-tree vSphere volumes will not get migrated successfully
-   - Workaround:
-      - Upgrade CSI driver with this fix.
+   - Workaround: This is being fixed in an upcoming version.
 7. When a Pod is rescheduled to a new node, there may be some lock contention which causes a delay in the volume getting detached from the old node and attached to the new node.
    - Impact: Rescheduled Pods remain in `Pending` state for an varying amount of time.
    - Workaround: Upgrade CSI driver to `v2.1.1`.
 8. When pod using a PVC is rescheduled to other node when metadatasyncer is down, fullsync might fail with error `Duplicated entity for each entity type in one cluster is found`.
    - Impact: CNS may hold stale volume metadata.
+   - Workaround: This is being fixed in an upcoming version.
+9. If there are no datastores present in any one of the datacenters in your vSphere environment, CSI file volume provisioning fails with `failed to get all the datastores` error.
+   - Impact: File volume provisioning keeps failing.
    - Workaround:
-      - Upgrade CSI driver with this fix.
+      - Remove the `ReadOnly` privilege on this datacenter for the user listed in the `vsphere-config-secret` secret. Refer to `vsphere-roles-and-privileges` section in the [prerequisites](../driver-deployment/prerequisites.md) page.
+      - Add a datastore to this datacenter
```

Comment:
Can you mention one of the workarounds is good enough.  Remove Read Permission from the datacenter or add datastore in the datacenter.

Reply:
I changed it, is this better?